package.json

```diff
@@ -9,7 +9,7 @@
     "ci:lint": "eslint --color --max-warnings 0 --quiet '{components,tests,utils,scripts}/**/*.js'",
     "ci:send-coverage": "codecov",
     "ci:test": "TZ=utc jest -c ./tests/jest.config.json --bail --ci --coverage --maxWorkers=2 --no-cache --silent",
-    "deploy:accp": "make prepare-site && now _site --public",
+    "deploy:accp": "BRANCH=$(git rev-parse --abbrev-ref HEAD); make prepare-site && now rm -y $BRANCH 2>/dev/null; now _site --public -n $BRANCH",
```

Comment:
Hmmm... will this work on Windows machines? Maybe it would be better to use the CI's env variable for the branch, in this case `TRAVIS_PULL_REQUEST_BRANCH` ? (See more at: https://docs.travis-ci.com/user/environment-variables/ )

Reply:
We could create a travis specific `now` account and find a way of skip authentication, but as I said this is meant to work locally. I didn't test on Windows, I since I assumed `make` wouldn't work there. Is there anybody here who can verify?

Reply:
Can you at least use the Travis CI's Environment variable instead of creating the 'BRANCH' one?

Reply:
Ahhh forget it. This is for the manual process... Oy oy, my brain was not getting it lol. 👍 Fine by me then :)